From 6d6361fe037318ef5a95c6136fda852276a021a7 Mon Sep 17 00:00:00 2001
From: Michael Krummsdorf <michael.krummsdorf@tq-group.com>
Date: Thu, 13 Oct 2016 09:49:54 +0200
Subject: [PATCH] dt:ls1021a: revert: add msi-parent for pcie nodes

original commit 51fb42e3a118b09eb32ece36be4b9c9fd9a25e9b
    arm/dt: ls1021a: add msi-parent for pcie nodes
Replace with mainline approach with separate msi-controller nodes.

Signed-off-by: Michael Krummsdorf <michael.krummsdorf@tq-group.com>
---
 arch/arm/boot/dts/ls1021a.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index b1d8d96..13149b7 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -141,8 +141,6 @@
 			      <0x0 0x1404000 0x0 0x2000>,
 			      <0x0 0x1406000 0x0 0x2000>;
 			interrupts = <GIC_PPI 9 (GIC_CPU_MASK_SIMPLE(2) | IRQ_TYPE_LEVEL_HIGH)>;
-			msi-controller;
-
 		};
 
 		msi1: msi-controller@1570e00 {
@@ -878,7 +876,6 @@
 					<0000 0 0 2 &gic GIC_SPI 188 IRQ_TYPE_LEVEL_HIGH>,
 					<0000 0 0 3 &gic GIC_SPI 190 IRQ_TYPE_LEVEL_HIGH>,
 					<0000 0 0 4 &gic GIC_SPI 192 IRQ_TYPE_LEVEL_HIGH>;
-			msi-parent = <&gic>;
 		};
 
 		pcie@3500000 {
@@ -902,7 +899,6 @@
 					<0000 0 0 2 &gic GIC_SPI 189 IRQ_TYPE_LEVEL_HIGH>,
 					<0000 0 0 3 &gic GIC_SPI 191 IRQ_TYPE_LEVEL_HIGH>,
 					<0000 0 0 4 &gic GIC_SPI 193 IRQ_TYPE_LEVEL_HIGH>;
-			msi-parent = <&gic>;
 		};
 	};
 };
